app/components/RegionDropdown/RegionDropdown.module.scss

```
@@ -7,13 +7,16 @@
   box-shadow: 0 0 0 2px var(--control-focus-ring);
 }
 
-$control-width: 260px; // Control width local variable for now
+$select-width: $space-full;
 
 .dropdown-container {
   position: relative;
   cursor: pointer;
-  width: $control-width;
+  width: $select-width;
```

minor: I'd probably just make this 100% - it's a bit easier to understand at face value compared to translating it to 'select-width' then 'space-full', it also probably doesn't need to come from a system value 💃 

```suggestion
  width: 100%;
```